package.json

```diff
@@ -33,7 +33,8 @@
     "README.md",
     "CHANGELOG.md",
     "LICENSE",
-    "npm-shrinkwrap.json"
+    "npm-shrinkwrap.json",
+    "scripts/*.mjs"
```

Comment:
simply include the whole `scripts` dir

Reply:
not sure if we also need to add this folder to the list of linted sources

Reply:
Hm, I might miss somewhere. `npx eslint scripts/install-chromedriver.mjs` worked but `npx eslint .` ignored the `scripts`.

Reply:
perhaps the global eslint config is only configured to verify .js files. We need to extend that and include mjs/cjs as well

Reply:
for now lets have the following pattern: `(mjs|cjs|js)`